wandb_workspaces/utils/validators.py

```diff
@@ -16,7 +16,10 @@ def validate_spec_version(
     spec: Dict[str, Any], *, expected_version: int
 ) -> Dict[str, Any]:
     actual_version = spec.get("version", -1)
-
+    
+    if actual_version <= 1:
+        return spec
+    
```

Comment:
I think this is breaking tests.  Why do you need this?

Reply:
okay i overthought my migration logic - changed to just update client spec version